step_command_cache_test.go

```diff
@@ -8,6 +8,66 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+func TestCacheMarshalJSON(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name string
+		c    Cache
+		want string
+	}{
+		{
```

Comment:
I think we'll also want to support:

```
cache: false
```

We could do that seperately, but with the overhead of an agent release and then a deploy of the agent to Namespace, it might be worth adding support for it now.